packages/jaeger-ui/src/utils/config/get-config.tsx

```diff
@@ -26,36 +26,47 @@ let haveWarnedDeprecations = false;
  * default config from `../../constants/default-config`.
  */
 const getConfig = memoizeOne(function getConfig() {
-  const getJaegerUiConfig = window.getJaegerUiConfig;
-  if (typeof getJaegerUiConfig !== 'function') {
-    if (!haveWarnedFactoryFn) {
-      // eslint-disable-next-line no-console
-      console.warn('Embedded config not available');
-      haveWarnedFactoryFn = true;
-    }
-    return { ...defaultConfig };
-  }
-  const embedded = getJaegerUiConfig();
-  if (!embedded) {
-    return { ...defaultConfig };
-  }
+  const embedded = getUiConfig();
+  const capabilities = getCapabilities();
   // check for deprecated config values
   if (Array.isArray(deprecations)) {
     deprecations.forEach(deprecation => processDeprecation(embedded, deprecation, !haveWarnedDeprecations));
     haveWarnedDeprecations = true;
   }
-  const rv = { ...defaultConfig, ...embedded };
+  const rv = { ...defaultConfig, ...embedded, ...capabilities };
   // mergeFields config values should be merged instead of fully replaced
   const keys = mergeFields;
   for (let i = 0; i < keys.length; i++) {
     const key = keys[i];
-    if (typeof embedded[key] === 'object' && embedded[key] !== null) {
+    if (embedded && typeof embedded[key] === 'object' && embedded[key] !== null) {
       rv[key] = { ...defaultConfig[key], ...embedded[key] };
     }
   }
   return rv;
 });
 
+function getUiConfig() {
+  const getJaegerUiConfig = window.getJaegerUiConfig;
+  if (typeof getJaegerUiConfig !== 'function') {
+    if (!haveWarnedFactoryFn) {
+      // eslint-disable-next-line no-console
+      console.warn('Embedded config not available');
+      haveWarnedFactoryFn = true;
+    }
+    return { ...defaultConfig };
```

Comment:
you never actually call `getJaegerUiConfig` to retrieve the values

Reply:
If there is no `embeded` value we should return `defaultConfig` else we return the result from `getJaegerUiConfig` call. (l.62)